pages/payment_page.py

```diff
@@ -1,28 +1,32 @@
 from pages.base_page import BasePage
+from selenium.webdriver.common.by import By
 import time
 
 class PaymentPage(BasePage):
+    PAY_TEXTBOX = (By.ID, 'pay-amount')
+    CLICK_OFF_PAY = (By.CLASS_NAME, 'pay-steps')
+    PAY_METHOD = (By.ID,'1119452')
```

Comment:
Is this ID autogenerated? I bet there's a good chance it breaks when they push an update to the page. Is there a class name you can use instead?